myuw/templates/handlebars/card/accounts/library.html

```diff
@@ -52,19 +50,27 @@ <h4 class="card-badge-label">You owe</h4>
                         </div>
                         <div class="pull-right">
                             <span class="card-badge-value text-danger">${{ formatPrice fines }}</span>
-			    &nbsp;
-                            <span class="card-badge-action pull-right"><a href="https://p.lib.washington.edu/payfines/" target="_blank" data-linklabel="Pay Library Fees">Pay fees</a></span>
+			  
+                            <span class="card-badge-action"><a href="https://p.lib.washington.edu/payfines/" target="_blank" data-linklabel="Pay Library Fees">Pay fees</a></span>
                         </div>
                     </div>
 		</li>
                 {{/if}}
+                <li>
+                    <div class="card-badge clearfix">
+                        <div class="pull-right">
+                            <span class="btn btn-default card-badge-action-btn pull-right"><a href="http://search.lib.uw.edu/account" target="_blank" data-linklabel="Your Library Account">Access library account</a></span>
+                        </div>
+                    </div>
+                </li>
+
 	    </ul>
         </div>
 
         {{#unless items_loaned }}
```

Comment:
This whole block of code inside the handlebar logic can be removed since we're showing the button in all cases.